vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -385,6 +385,32 @@ void FrameworkElementViewManager::StartTransformAnimation(
   uielement.StartAnimation(expression);
 }
 
+folly::dynamic FrameworkElementViewManager::GetCommands() const
+{
+  // Don't update SetFocus/Blur commands in subclass, otherwise commands may not match with what defined in js side.
+	auto commands = Super::GetCommands();
+	commands.update(folly::dynamic::object
+	("SetFocus", static_cast<std::underlying_type_t<FocusCommand>>(FocusCommand::SetFocus))
+	);
+	commands.update(folly::dynamic::object
+	("Blur", static_cast<std::underlying_type_t<FocusCommand>>(FocusCommand::Blur))
+	);
+	return commands;
+}
+
+void FrameworkElementViewManager::DispatchCommand(XamlView viewToUpdate, int64_t commandId, const folly::dynamic& commandArgs)
+{
+  // UWP doesn't have the blur concept. Here we use FocusState::Pointer to move away to keyboard focused visual.
+	if (viewToUpdate)
+	{
+		auto focusCommand = (static_cast<FocusCommand>(commandId));
+		if (focusCommand == FocusCommand::SetFocus)
+			winrt::FocusManager::TryFocusAsync(viewToUpdate, winrt::FocusState::Programmatic);
+		else if (focusCommand == FocusCommand::Blur)
+			winrt::FocusManager::TryFocusAsync(viewToUpdate, winrt::FocusState::Pointer);		
+	}
```

Comment:
PR #2687 now out for review is updating the implementation of Blur in the TextInputViewManager so it doesn't do anything if they TextBox doesn't have focus. Might be good to integrate that functionality here now.
[https://github.com/microsoft/react-native-windows/pull/2687](https://github.com/microsoft/react-native-windows/pull/2687)

Reply:
I didn't see any conflicts for these two changes, and I hope it would be merged correctly. I will pay attention to it.

---
In reply to: [297795330](https://github.com/microsoft/react-native-windows/pull/2682#discussion_r297795330) [](ancestors = 297795330)

Reply:
Sorry, my concern was that the fix in that PR should probably be applied here as well. That PR is fixing something that this one might then reintroduce for other elements.

---
In reply to: [297802284](https://github.com/microsoft/react-native-windows/pull/2682#discussion_r297802284) [](ancestors = 297802284,297795330)

Reply:
Agreed. Fixed

---
In reply to: [297835286](https://github.com/microsoft/react-native-windows/pull/2682#discussion_r297835286) [](ancestors = 297835286,297802284,297795330)